### PR TITLE
Fixing nested model update

### PR DESCRIPTION
**Pull request by xavijam**
Related with: https://github.com/CartoDB/cartodb/issues/12386

cc @nobuti

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/components/form-components/editors/radio/radio.js [hunk 1]
```diff
@@ -80,6 +80,8 @@ Backbone.Form.editors.Radio = Backbone.Form.editors.Radio.extend({
     });
   },
 
+  _destroyBinds: function () {},
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/components/form-components/editors/radio/radio.js]: This overwrites the parent method, is it what we want?

Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/components/form-components/editors/radio/radio.js]: It doesn't make sense sorry, it should "work" now.

File: lib/assets/core/javascripts/cartodb3/components/form-components/editors/radio/radio.js [hunk 2]
```diff
@@ -84,6 +84,6 @@ Backbone.Form.editors.Radio = Backbone.Form.editors.Radio.extend({
     if (this._helpTooltip) {
       this._helpTooltip.clean();
     }
-    Backbone.Form.editors.Base.prototype.remove.call(this);
+    Backbone.Form.editors.Radio.prototype.remove.call(this);
```
Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/components/form-components/editors/radio/radio.js]: There is something fun here. If we set this ^^^, a maximum call stack error appears. So "using" Base remove function, it requires `_destroyBinds` method. 💃

**Comment by nobuti** [comment]
I can see any flaws in staging, it's working 200% better than before ✨

**Comment by xavijam** [comment]
Nice!, thanks for giving a try man.

**Pull request merged**